src/run.ts

```diff
@@ -78,7 +82,38 @@ export async function runPublish({
   createGithubReleases,
   cwd = process.cwd(),
 }: PublishOptions): Promise<PublishResult> {
-  let octokit = github.getOctokit(githubToken);
+  const octokit = new (GitHub.plugin(throttling))(
+    getOctokitOptions(githubToken, {
+      throttle: {
+        onRateLimit: (retryAfter, options: any, octokit, retryCount) => {
+          octokit.log.warn(
```

Comment:
We don't use `octokit.log` anywhere else. On top of that I see that `actokit.log.info` is basically noop (can be checked [here](https://github.com/octokit/core.js/tree/928dd9fb0cb81d4798b3127b423085f4b6bfaaa6#logging)). Do u have any opinions on how logging should be done in the github actions in general and here?

Reply:
> It would be better to take a holistic pass at logging in general in a separate branch to make sure it's consistent and works as intended across the board, so I would strip out the octokit logging for now, and/or use the established pattern for logging.

This sounds like a good plan. Let's just replace the octokit logging with plain console-based functions and create a ticket to revamp logging in the future.